### PR TITLE
Update diskThreshold to availableDiskThreshold

### PR DESCRIPTION
**Pull request by NehaNaithani**
- Does the iofog.org or README.md documentation need to change?
  - [ ] Yes 
  - [ ] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -104,7 +104,7 @@
     "xss-clean": "0.1.1"
   },
   "devDependencies": {
-    "acorn": "6.1.1",
+    "acorn": "^7.1.1",
```
Comment (Pixcell) [inline package.json]: Dependencies need to be fixed version (Remove ^)

File: package.json [hunk 2]
```diff
@@ -114,7 +114,7 @@
     "eslint-config-google": "0.12.0",
     "mocha": "5.2.0",
     "newman": "4.5.7",
-    "nyc": "14.1.1",
+    "nyc": "^15.0.0",
```
Comment (Pixcell) [inline package.json]: Same here

**Review by Pixcell — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (NehaNaithani) [reply package.json]: updated

File: package.json (hunk 2 above)
Comment (NehaNaithani) [reply package.json]: updated

**Pull request merged**